### PR TITLE
Move documentation from Wiki to GitHub

### PR DESCRIPTION
**Pull request by zbynek**
See https://issues.jenkins-ci.org/browse/WEBSITE-637

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Changelog
+
+## Version 1.20 (Jul 29, 2019)
+
+-   [JENKINS-33217](https://issues.jenkins-ci.org/browse/JENKINS-33217) Log a warning when additional parameters are passed into \`DescribableModel\` objects, such as Pipeline steps, since these parameters are currently being ignored.
+-   Internal - Update parent pom ([PR-46](https://github.com/jenkinsci/structs-plugin/pull/46){.external-link}) and fix unit tests ([PR-50](https://github.com/jenkinsci/structs-plugin/pull/50){.external-link})
```
Comment (car-roll) [inline CHANGELOG.md]: You need to strip all the `{.external-link}` tags. The wiki markdown converter unfortunately leaves all of those in place.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+## Version 1.2.0 (Jun 17, 2016)
+
+-  ℹ️ Added method to query deprecated methods ([PR #5](https://github.com/jenkinsci/structs-plugin/pull/5))
+-  ❌ Improve diagnostics for mismatched types ([JENKINS-34070](https://issues.jenkins-ci.org/browse/JENKINS-34070))
+-  ❌ Prevent recursions in the DescribableModel.toString() method ([PR #3](https://github.com/jenkinsci/structs-plugin/pull/3), related to [JENKINS-32925](https://issues.jenkins-ci.org/browse/JENKINS-32925))
+
+## Version 1.1.1 (Jun 16, 2016)
+
+-  ❌ Fix URL to the plugin's Wiki page in order to get it listed in Jenkins Update Center again ([JENKINS-JENKINS-35918](https://issues.jenkins-ci.org/browse/JENKINS-JENKINS-35918))
```
Comment (car-roll) [inline CHANGELOG.md]:
```suggestion
-  ❌ Fix URL to the plugin's Wiki page in order to get it listed in Jenkins Update Center again ([JENKINS-35918](https://issues.jenkins-ci.org/browse/JENKINS-35918))
```

File: README.md [hunk 3]
```diff
@@ -1 +1,61 @@
-See [wiki page](https://wiki.jenkins-ci.org/display/JENKINS/Structs+plugin) for further details
+# Structs plugin
+[![Jenkins Plugins](https://img.shields.io/jenkins/plugin/v/structs)](https://plugins.jenkins.io/structs)
```
Comment (car-roll) [inline README.md]:
```suggestion
[![Jenkins Plugins](https://img.shields.io/jenkins/plugin/v/structs)](https://plugins.jenkins.io/structs)
[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/structs-plugin?label=changelog)](https://github.com/jenkinsci/structs-plugin/blob/master/CHANGELOG.md)
[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/structs?color=blue)](https://plugins.jenkins.io/structs)
```
You might want to double check if my links are correct

**Comment by zbynek** [comment]
@car-roll thanks! I applied your suggestions.

**Comment by oleg-nenashev** [comment]
@car-roll Hi. Would it be possible to get it released?

**Comment by car-roll** [comment]
@oleg-nenashev I think so. Let me just check if there is another PR we can get in.

**Comment by car-roll** [comment]
@oleg-nenashev I can go ahead a cut a release for this. I was just wondering is there any particular reason we wanted this released now with just wiki migration?

**Comment by timja** [comment]
Shall we release this @car-roll?

it's the most popular plugin that the migration hasn't been completed for

**Comment by car-roll** [comment]
@timja 1.21 has now been released

**Pull request merged**